frontend/src/services/SubmitRecordService.js

```diff
@@ -32,4 +38,19 @@ export default class SubmitRecordService {
       );
     });
   }
+  static submit_duration(username, start_time, end_time) {
```

Comment:
- [ ] (Optional) I would suggest to rename duration to interval, this is optional to you. For explanation please check my previous comment